tensorflow/core/kernels/mkl_fused_batch_norm_op.cc

```diff
@@ -262,6 +262,7 @@ class MklFusedBatchNormOp : public OpKernel {
     }
 
     void MklCreateInputLayout(OpKernelContext* context) {
+      const Tensor& input = MklGetInput(context, 0);
```

Comment:
Just curious, why do we have this line? This local variable isn't used anywhere in the function.

Reply:
Should not be there. See more comment below.